content/en/blog/kotlin-gradle-projects.md

```diff
@@ -9,13 +9,16 @@ author:
 
 Did you know that gRPC Java now has out of box support for Kotlin projects built with Gradle? [Kotlin](https://kotlinlang.org/) is a modern, statically typed language developed by JetBrains that targets the JVM and Android. It is generally easy for Kotlin programs to interoperate with existing Java libraries. To improve this experience further, we have added support to the [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin/releases) so that the generated Java libraries are automatically picked up by Kotlin. You can now add the protobuf-gradle-plugin to your Kotlin project, and use gRPC just like you would with a typical Java project.
 
-<!--more-->
+{{% alert title="Native Kotlin support" color="success" %}}
+Looking for native Kotlin support of gRPC? See [Kotlin, meet gRPC]({{<relref "kotlin-meet-grpc">}}).
+{{% /alert %}}
```

Comment:
@jamesward : are you ok with the formulation of this alert?

Reply:
The word "native" can be something a different (non-JVM stuff) so I wonder if there is another word we can use.  Or reword this somehow.

Reply:
That was my feeling too, which is why I wanted your input. Any suggested reformulations?

Reply:
We could just drop the "native" word.

Reply:
Yes, we could drop it from the title, but IMHO it seems odd to drop it from the alert text, which would then read "Looking for Kotlin support for gRPC?" That is what the blog post is about -- Kotlin support via gRPC Java.

Would you be willing to keep "native" in the alert text? If not, I'm open to other formulations. Thanks.

Reply:
For the record: after some more thought, @jamesward is ok with keeping "native" in both the title and the alert text (as confirmed out-of-band).